app/app/src/main/kotlin/com/hedvig/app/feature/profile/ui/payment/PaymentDestination.kt

```diff
@@ -70,23 +73,29 @@ fun PaymentDestination(
 ) {
   val context = LocalContext.current
   val uiState by viewModel.uiState.collectAsStateWithLifecycle()
-  PaymentScreen(
-    uiState = uiState,
-    locale = Locale.ENGLISH,
-    navigateUp = onBackPressed,
-    onChangeBankAccount = {
-      context.startActivity(
-        connectPayinIntent(
-          context,
-          PaymentType.TRUSTLY,
-          Market.SE,
-          false,
-        ),
+
+  AnimatedContent(targetState = uiState.isLoading) { loading ->
```

Comment:
This by default does a slight scale transition (from 92% to 100% size) and a fade.
There's also just `CrossFade` which only does the fade without the scale effect.
I don't have a personal opinion on which one we'd prefer, just letting you know what easy options we got so you can pick the one you feel like is best for simple transitions like these.

Reply:
Good to know, we can check with William later! I dont have any strong opinions, think both look good :)

Reply:
Yeah, then just keep it and we can discuss in the future